mxcubeweb/routes/lims.py

```diff
@@ -61,7 +62,24 @@ def get_proposal():
         return jsonify({"Proposal": app.lims.get_proposal_info()})
 
     def run_get_result_script(script_name, url):
-        return check_output(["node", script_name, url], close_fds=True)
+        node_path=shutil.which("node")
```

Comment:
The three methods below can be removed entierly :) :
- run_get_result_script(script_name, url)
- result_file_test(prefix):
- apply_template(name, data)